esp/services/ws_smc/ws_smcService.cpp

```diff
@@ -141,6 +141,7 @@ void CWsSMCEx::init(IPropertyTree *cfg, const char *process, const char *service
 
     xpath.setf("Software/EspProcess[@name=\"%s\"]/EspService[@name=\"%s\"]/ActivityInfoCacheSeconds", process, service);
     activityInfoCacheSeconds = cfg->getPropInt(xpath.str(), DEFAULTACTIVITYINFOCACHETIMEOUTSECOND);
+    querySDS().setConfigOpt("Client/@LogConnection", "1");
```

Comment:
Should probably be picked up as configuration setting and defaulted to off.
"true" vs "1" would make more sense also.